src/tests/Parser/Brewmaster/DamageTaken.test.js

```diff
@@ -1,10 +1,11 @@
 import DamageTaken from 'Parser/BrewmasterMonk/Modules/Core/DamageTaken';
 import { events, processEvents } from './Fixtures/SimpleFight';
 
-const damageTaken = new DamageTaken();
+let damageTaken = null;
```

Comment:
This could be put in the `describe` function, and can be just `let damageTaken;` without value